packages/Angular/explorer-core/src/lib/list-component/list-component.ts

```diff
@@ -0,0 +1,200 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Metadata, RunView } from '@memberjunction/core';
+import { ListEntity } from '@memberjunction/core-entities'; 
+import { NotificationService } from "@progress/kendo-angular-notification";
+
+export class List {
```

Comment:
You dont need to create a custom class as the `ListEntity` class has these properties and is more appropriate to use here